packages/core/src/Select/BaseSelect.tsx

```diff
@@ -169,7 +173,7 @@ const SelectOptionComponent: React.FC<SelectOption> = ({
       selected={selected && selectMarker === 'background'}
       ref={ref}
     >
-      {item}
+      <Item>{item}</Item>
```

Comment:
What issue does this solve? This should perhaps be fixed elsewhere...

Reply:
@Tigge The `<SelectItem>` container renders an `<li>` that has this style `flex flex-column`. Setting `overflow: hidden` and `text-overflow: ellipsis` on this container has no effect. We have to wrap it in another element, in this instance a `<span>` so that we can center hide the overflow and also show the text ellipsis as intended.

Reply:
a `white-space: nowrap;` on the container perhaps?

Reply:
@Tigge still has no effect if applied on the `li` container. Only works if we wrap the li in this instance a `span`